lib/config.js

```diff
@@ -20,7 +20,8 @@ var fs = require("fs"),
     assign = require("object-assign"),
     debug = require("debug"),
     yaml = require("js-yaml"),
-    userHome = require("user-home");
+    userHome = require("user-home"),
+    isAbsolutePath = require("path-is-absolute");
```

Comment:
I hated adding this, but if we want to support windows, we unfortunately can't just check for /. Node 0.12 has a native `path.isAbsolute()`, but contribution guidelines says to still support 0.10, so...